pages/content/it-admins.mdx

```diff
@@ -25,7 +25,10 @@ If you are unable to whitelist all subdomains, please whitelist the following:
 - `sprig.hackclub.com`
 
 ## GitHub
-Besides the typical HTTP(S) ports, ports 22 and 9418 need to be open for SSH and Git respectively.
+Besides the typical HTTP(S) ports, ports 22 and 9418 need to be open for SSH and Git respectively. 
+  
+## Inspect window in Chrome
+The Inspect window in Chrome is required for any web development. The Inspect window in Chrome allows for web pages to be debugged and improved. It allows for the creation of appealing designs and more intuitive interfaces. 
```

Comment:
```suggestion
Browser developer tools (eg. Inspect Element) are required for web development projects, so that sites can be debugged and styled.
```